<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIChatRoom</class>
 <widget class="QWidget" name="UIChatRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UIChatRoom</string>
  </property>
  <widget class="QPushButton" name="button_sendMseeage">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>600</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cleanText">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>570</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清屏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_talk">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>讨论</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_brow">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>570</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>表情</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_studentList">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>102</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>学员</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_notes">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>570</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>消息记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_proclamation">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>公告</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>301</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="personList" name="student_list"/>
    </item>
    <item>
     <widget class="QWidget" name="proclamationWidget" native="true">
      <widget class="QTextBrowser" name="text_proclamation">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>299</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>299</width>
         <height>91</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>请在此输入新公布内容（最多支持一百字）</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_sendMseeage_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>+发布公告</string>
       </property>
      </widget>
      <widget class="QPushButton" name="button_sendMseeage_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>238</x>
         <y>100</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>发布</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="msgRecord" native="true">
      <widget class="myLabel" name="timeShow">
       <property name="geometry">
        <rect>
         <x>115</x>
         <y>5</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_1">
       <property name="geometry">
        <rect>
         <x>95</x>
         <y>5</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>5</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
      <widget class="AnimatedTextBrowser" name="talkRecord">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>38</y>
         <width>299</width>
         <height>501</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="recordLabel">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>5</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(242, 242, 242);</string>
       </property>
       <property name="text">
        <string>消息记录</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_3">
       <property name="geometry">
        <rect>
         <x>255</x>
         <y>5</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
      <widget class="QCalendarWidget" name="timeWidget">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>5</y>
         <width>297</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(160, 159, 164);
alternate-background-color: rgb(198, 185, 166);
selection-background-color: rgb(0, 220, 161);</string>
       </property>
       <property name="selectionMode">
        <enum>QCalendarWidget::SingleSelection</enum>
       </property>
       <property name="horizontalHeaderFormat">
        <enum>QCalendarWidget::ShortDayNames</enum>
       </property>
       <property name="verticalHeaderFormat">
        <enum>QCalendarWidget::NoVerticalHeader</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="AnimatedTextBrowser" name="text_talk">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="AnimatedTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>600</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="placeholderText">
    <string>请在此输入</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AnimatedTextEdit</class>
   <extends>QTextEdit</extends>
   <header>animatedtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>AnimatedTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>animatedtextbrowser.h</header>
  </customwidget>
  <customwidget>
   <class>personList</class>
   <extends>QListWidget</extends>
   <header>personlist.h</header>
  </customwidget>
  <customwidget>
   <class>myLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
